public/app/features/alerting/unified/api/stateHistoryApi.ts

```diff
@@ -22,6 +30,14 @@ export const stateHistoryApi = alertingApi.injectEndpoints({
           });
         }
 
+        if (previous) {
+          params.previous = previous;
+        }
+
+        if (current) {
+          params.current = current;
+        }
```

Comment:
I think it should be fine to just pass `previous` and `current` as is in line `24`